Makefile

```diff
@@ -42,6 +42,7 @@ LDLIBS = $(TEST_LIBS)
 SUDO =
 
 CCTZ_LIB = libcctz.a
+CCTZ_STATIC_LIB = $(CCTZ_LIB)
```

Comment:
Let's just leave CCTZ_LIB as it is then.  That is, forget about CCTZ_STATIC_LIB ... just introduce CCTZ_SHARED_LIB.